exporter/clickhouseexporter/README.md

```diff
@@ -1,117 +1,255 @@
+
+```clickhouse
+WITH
+    '391dae938234560b16bb63f51501cb6f' as trace_id,
+    (SELECT min(Start) FROM otel_traces_trace_id_ts WHERE TraceId = trace_id) as start,
+    (SELECT max(End) + 1 FROM otel_traces_trace_id_ts WHERE TraceId = trace_id) as end
+SELECT Timestamp as log_time,
+       TraceId,
+       SpanId,
+       ParentSpanId,
+       SpanName,
+       SpanKind,
+       ServiceName,
+       Duration,
+       StatusCode,
+       StatusMessage,
+       toString(SpanAttributes),
+       toString(ResourceAttributes),
+       toString(Events.Name),
+       toString(Links.TraceId)
+FROM otel_traces
+WHERE TraceId = trace_id
+  AND Timestamp >= start
+  AND Timestamp <= end
+Limit 100;
+```
+
+- Find spans is error.
+
+```clickhouse
+SELECT Timestamp as log_time,
+       TraceId,
+       SpanId,
+       ParentSpanId,
+       SpanName,
+       SpanKind,
+       ServiceName,
+       Duration,
+       StatusCode,
+       StatusMessage,
+       toString(SpanAttributes),
+       toString(ResourceAttributes),
+       toString(Events.Name),
+       toString(Links.TraceId)
+FROM otel_traces
+WHERE ServiceName = 'clickhouse-exporter'
+  AND StatusCode = 'STATUS_CODE_ERROR'
+  AND Timestamp >= NOW() - INTERVAL 1 HOUR
+Limit 100;
+```
+
+- Find slow spans.
+
+```clickhouse
+SELECT Timestamp as log_time,
+       TraceId,
+       SpanId,
+       ParentSpanId,
+       SpanName,
+       SpanKind,
+       ServiceName,
+       Duration,
+       StatusCode,
+       StatusMessage,
+       toString(SpanAttributes),
+       toString(ResourceAttributes),
+       toString(Events.Name),
+       toString(Links.TraceId)
+FROM otel_traces
+WHERE ServiceName = 'clickhouse-exporter'
+  AND Duration > 1 * 1e9
+  AND Timestamp >= NOW() - INTERVAL 1 HOUR
 Limit 100;
 ```
 
 ## Performance Guide
 
-A single clickhouse instance with 32 CPU cores and 128 GB RAM can handle around 20 TB (20 Billion) logs per day, 
+A single clickhouse instance with 32 CPU cores and 128 GB RAM can handle around 20 TB (20 Billion) logs per day,
 the data compression ratio is 7 ~ 11, the compressed data store in disk is 1.8 TB ~ 2.85 TB,
 add more clickhouse node to cluster can increase linearly.
 
-The otel-collector with `otlp receiver/batch processor/clickhouse tcp exporter` can process 
+The otel-collector with `otlp receiver/batch processor/clickhouse tcp exporter` can process
 around 40k/s logs entry per CPU cores, add more collector node can increase linearly.
 
 ## Configuration options
 
 The following settings are required:
 
-- `dsn` (no default): The ClickHouse server DSN (Data Source Name), for example `tcp://127.0.0.1:9000?username=user&password=qwerty&database=default`
-   For tcp protocol reference: [ClickHouse/clickhouse-go#dsn](https://github.com/ClickHouse/clickhouse-go#dsn).
-   For http protocol reference: [mailru/go-clickhouse/#dsn](https://github.com/mailru/go-clickhouse/#dsn).
+- `dsn` (no default): The ClickHouse server DSN (Data Source Name), for
```

Comment:
Not for this PR, but why this option must be `dsn` and not `endpoint` as all other exporter configurations have?

Reply:
Good advice, I will submit another PR use `endpoint` for consistent.
the `dsn` can be split to `endpoint` `database` `params`